### PR TITLE
Add new JSON tree library to be able to bundle Sheikah

### PR DESCRIPTION
**Pull request by gabaldon**
This pr adds a new JSON tree library to be able to bundle Sheikah.

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -181,22 +183,22 @@ function createWindow() {
...
+  //       submenu: [
+  //         { label: 'Reload', accelerator: 'CmdOrCtrl+R', click: () => {} },
+  //         { label: 'ZoomOut', accelerator: 'CmdOrCtrl+-', click: () => {} },
+  //         { label: 'ZoomIn', accelerator: 'CmdOrCtrl+Plus', click: () => {} },
+  //       ],
+  //     },
+  //   ])
+
+  //   Menu.setApplicationMenu(menu)
+  // }
```
Comment (aesedepece) [inline src/background.js:201]: Is this within the scope of this PR?

Comment (Tommytrg) [reply src/background.js:201]: AFAIK, this should be a draft and this is useful to debug the build

**Comment by Tommytrg** [comment]
Could you add the changes from [my branch build-2](https://github.com/Tommytrg/sheikah/tree/build-2)?

**Comment by gabaldon** [comment]
duplicated in #1315

**Pull request closed without merging**